rllib/evaluation/collectors/agent_collector.py

```diff
@@ -178,6 +178,11 @@ def add_action_reward_next_obs(self, input_values: Dict[str, TensorType]) -> Non
             Must contain keys:
                 SampleBatch.ACTIONS, REWARDS, DONES, and NEXT_OBS.
         """
+
+        # TODO (Kourosh): Error out when one of the keys are missing.
```

Comment:
Do u wanna also do the todos since we are here?

Reply:
It will save us time later imo.

Reply:
Sure thing. Let me do that in a follow up PR to keep the scope of this one smaller! Someone told me I should do that sort of thing

Reply:
Oh actually I think it should be done here in this PR. Because it was in this PR that we realized these errors must be raised. So we gotta double check this assumption by making sure that no error is raised on the current ci. Also you need to make sure that there is a test that checks if these errors are raised if the conditions are met.

Reply:
I've played around with this now and I strongly recommend that we don't go further down this path for now.
--> https://github.com/ray-project/ray/pull/30024#issuecomment-1309830761